### DISCUSSION TITLE
How are the "screenshots" rendered in the MUI manual?

### DISCUSSION THREAD

**Discussion by loopj** [olikraus/u8g2#1927] [Q&A] [answered]
I'm curious to know how the yellow "screenshots" are rendered, are you using some kind of emulator? Would love to take some of my own.

**Comment by olikraus** [comment, marked as answer]
For https://github.com/olikraus/u8g2/wiki/u8g2reference and https://github.com/olikraus/u8g2/wiki/muimanual there is indeed an sdl emulation https://github.com/olikraus/u8g2/blob/master/sys/sdl/mui_ref/main.c for the creation of the animations.

This in turn has a screenshot function
https://github.com/olikraus/u8g2/blob/7395c6857f3ade38a636b67365427178ad528dfa/sys/sdl/mui_ref/main.c#L871

which is based on the 
https://github.com/olikraus/u8g2/blob/7395c6857f3ade38a636b67365427178ad528dfa/csrc/u8x8.h#L814
function which is a unix extention: https://github.com/olikraus/u8g2/tree/master/sys/bitmap/common

**Reply by loopj** [reply]
Very helpful, thank you!
